spec/models/mixins/ownership_mixin_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe "Ownership" do
+  describe "reporting" do
+    describe "searching by current ldap group" do
+      let(:exp)    { {"="=>{"field"=>"Vm-owned_by_current_ldap_group", "value"=>"true"}} }
+      let(:report) { MiqReport.new.tap {|r| r.db = 'Vm'} }
+      let(:search_opts) { { :filter => MiqExpression.new(exp), :per_page => 20, :named_scope => :with_miq_group } }
+
+      before do
+        @miq_group = FactoryGirl.create :miq_group
+        User.current_user = FactoryGirl.create :user,
+                                               :current_group => @miq_group,
+                                               :miq_groups => [@miq_group]
+        expect(User).to receive(:server_timezone).and_return("UTC")
+
+        vm_with_group = FactoryGirl.create :vm_vmware, :miq_group_id => @miq_group.id
+        vm_no_group   = FactoryGirl.create :vm_vmware
+      end
+
+      it "returns results only part of the miq_group" do
+        vm_ids = report.paged_view_search(search_opts).first.map(&:id)
+        vms    = Vm.where(:id => vm_ids)
+        expect(vms.all? {|vm| vm.miq_group_id == @miq_group.id }).to be true
```

Comment:
Can you add a test for vms that are in a group but not *this* particular group?

Additionally, from a test structure perspective, I would expect that the vm_with_group would be in the test itself since it links the setup to the expectation and makes it clearer to me what the result should look like...something like

```ruby
expect(vms).to match_array [vm_with_group]
```